src/roles/nuh-deploy/templates/haproxy.cfg.j2

```diff
@@ -0,0 +1,26 @@
+frontend vsd_7443 *:7443
```

Comment:
I do not see this file being used in nuh-deploy? Is it used someplace else? Can we move it if that is the case.

Reply:
This was my failed attempt to manually configure haproxy on the NUH.  If you recall, I had a huge amount of trouble with this.  It turned out that the NUH just doesn't support stats-out yet.  I'll remove the file as it is no longer needed.